src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala

```diff
@@ -288,6 +288,7 @@ class PodsResourceTest extends AkkaUnitTest with Mockito {
       val response = f.podsResource.create(podSpecJsonWithContainerNetworking.getBytes(), force = false, f.auth.request)
       response.getStatus shouldBe 422
       response.getEntity.toString should include(NetworkValidationMessages.NetworkNameMustBeSpecified)
+      println(response.getEntity.toString)
```

Comment:
I'm not sure why we are printing in a test... I'm ok with it as long as we are asserting (which we are)

Reply:
Good catch. That is a left over from debugging.